addon/components/frost-select-dropdown.js

```diff
@@ -523,7 +529,7 @@ export default Component.extend({
 
         case TAB:
           this.get('onClose')()
-          return
+          return // eslint-disable-line no-useless-return
```

Comment:
Remove the `return`

Reply:
I will open an issue for us to refactor this code

Reply:
Issue opened to track this: https://github.com/ciena-frost/ember-frost-core/issues/506